### PR TITLE
♻️ Maintenance: mypy dynamic-sidecar

### PR DESCRIPTION
**Pull request by GitHK**
## What do these changes do?

- ♻️ removed duplicate models form `sicmore-sdk` and `dask-task-models-library`

## Related issue/s

- https://github.com/ITISFoundation/osparc-simcore/issues/6115
## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

File: packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import re
+
+from models_library.basic_regex import PROPERTY_KEY_RE
+from pydantic import ConstrainedStr
+
+
+class PortKey(ConstrainedStr):
+    regex = re.compile(PROPERTY_KEY_RE)
```
Comment (sanderegg) [inline packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py:8]: What is the difference with ```ServicePortKey``` in models_library.services_types.py ?

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py [hunk 2]
```diff
@@ -106,7 +106,6 @@ async def get_containers_activity(
 
     container_name = inactivity_command.service
 
-    inactivity_response: str | None = None
     try:
         inactivity_response = await run_command_in_container(
```
Comment (sanderegg) [inline services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:110]: what happens if inactivity_response is None?

**Review by sanderegg — APPROVED** [review]
thanks!

**Review by matusdrobuliak66 — APPROVED** [review]

File: services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py (hunk 2 above)
Comment (GitHK) [reply services/dynamic-sidecar/src/simcore_service_dynamic_sidecar/api/containers.py:110]: The `inactivity_response` is always a string when this succeeds.
Also below code will also handle the `None`, which will never happen.

File: packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py (hunk 1 above)
Comment (GitHK) [reply packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py:8]: None apparently. Let's try to reuse it. I will remove this one.

Comment (GitHK) [reply packages/simcore-sdk/src/simcore_sdk/node_ports_v2/types.py:8]: @sanderegg I have also removed it from the dask-task-models-library

**Review by pcrespov — APPROVED** [review]
thx. Please check my comment on the CI

**Pull request merged**